api/src/main/java/com/epam/pipeline/manager/preference/SystemPreferences.java

```diff
@@ -130,6 +130,21 @@
         "storage.temp.credentials.duration", 3600, DATA_STORAGE_GROUP, isGreaterThan(0));
     public static final IntPreference STORAGE_MOUNTS_PER_GB_RATIO = new IntPreference(
             "storage.mounts.per.gb.ratio", null, DATA_STORAGE_GROUP, isNullOrGreaterThan(0));
+    public static final BooleanPreference DEFAULT_USER_DATA_STORAGE_ENABLED =
+        new BooleanPreference("storage.user.home.auto", false, DATA_STORAGE_GROUP, pass);
+    public static final StringPreference DEFAULT_USER_DATA_STORAGE_TEMPLATE =
+        new StringPreference("user.default.storage.template",
```

Comment:
Could you rename preference to `storage.user.home.template` and make it an 'ObjectPreference' to enable json validation on the GUI?

Reply:
Done